sktime/classification/_oscnn.py

```diff
@@ -0,0 +1,201 @@
+from sklearn.metrics import accuracy_score
+import numpy as np
+import torch
```

Comment:
no torch imports should be a tmodule level - have a look how this is isolated inside the class methods in the other classes that use `torch`.

Reply:
ok, thank you for stating it